CHANGELOG.md

```diff
@@ -4,6 +4,44 @@ This file contains notable changes (e.g. breaking changes, major changes, etc.)
 
 This file was introduced starting Kani 0.23.0, so it only contains changes from version 0.23.0 onwards.
 
+## [0.53.0]
+
+### Major Changes
```

Comment:
Should we also add the checking of memory initialization to the major changes list?

Reply:
Added.